features/steps/step_checking_status.py

```diff
@@ -1,4 +1,4 @@
-from behave import then, given
+from behave import then, step
 from selenium import webdriver
 
 driver = webdriver.Chrome
```

Comment:
Do we need it in the step file ?

Reply:
I would prefer to switch context instead of passing it to steps